.env.example

```
@@ -403,12 +401,16 @@ NEUROLINK_MAX_TOOLS_PER_PROVIDER=5
 # Config File Location
 NEUROLINK_CONFIG_FILE=./neurolink.config.json
 
-# Summarization (Conversation Memory)
-NEUROLINK_SUMMARIZATION_ENABLED=false
-NEUROLINK_SUMMARIZATION_THRESHOLD_TURNS=20
-NEUROLINK_SUMMARIZATION_TARGET_TURNS=10
-NEUROLINK_SUMMARIZATION_PROVIDER=google-ai
-NEUROLINK_SUMMARIZATION_MODEL=gemini-2.5-flash
+# Summarization (Conversation Memory) - TOKEN-BASED MEMORY ENABLED BY DEFAULT
+# Token-based memory is enabled by default and uses 80% of each model's context window
+NEUROLINK_SUMMARIZATION_ENABLED=true             # Enable summarization (default: true)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)